plugin/wavesurfer.minimap.js

```diff
@@ -94,7 +94,7 @@ WaveSurfer.Minimap = WaveSurfer.util.extend({}, WaveSurfer.Drawer, WaveSurfer.Dr
 
     bindWaveSurferEvents: function () {
         var my = this;
-        this.wavesurfer.on('ready', this.render.bind(this));
+        this.wavesurfer.on('ready', my.render());
```

Comment:
@katspaugh yes this.render(); will work. I had already updated it to that in the preview code but didn't in this pull. I'll do that now.